### PR TITLE
Show rating scale

### PR DESCRIPTION
**Pull request by nikhilgeo**
PR for issue #81

I have added the rating scale to 
movie.py line no: 11
anime.py line no: 15
book.py line no: 25

### CODE REVIEW

**Comment by swapagarwal** [comment]
IMDb ratings are on a scale of 10.
Can you verify the others as well?

**Comment by nikhilgeo** [comment]
Copy that !!
I will verify and update..

File: modules/src/anime.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def process(input, entities):
 
         template = TextTemplate()
         template.set_text('Title: ' + data[0]['title'] + '\nSynopsis: ' + data[0]['synopsis'])
-        template.set_post_text('\nCommunity Rating: ' + str(round(data[0]['community_rating'], 2)) + '\nStatus: ' + data[0]['status'])
+        template.set_post_text('\nCommunity Rating: ' + str(round(data[0]['community_rating'], 2)) +'/5'+ '\nStatus: ' + data[0]['status'])
```
Comment (swapagarwal) [inline modules/src/anime.py]: proper spacing would be great!
`+ ' / 5' +`

**Comment by nikhilgeo** [comment]
IMDB is on 10, rest all are under 5.

I have corrected the spacing also...

File: modules/src/anime.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def process(input, entities):
 
         template = TextTemplate()
         template.set_text('Title: ' + data[0]['title'] + '\nSynopsis: ' + data[0]['synopsis'])
-        template.set_post_text('\nCommunity Rating: ' + str(round(data[0]['community_rating'], 2)) + '\nStatus: ' + data[0]['status'])
+        template.set_post_text('\nCommunity Rating: ' + str(round(data[0]['community_rating'], 2)) +' /5'+ '\nStatus: ' + data[0]['status'])
```
Comment (swapagarwal) [inline modules/src/anime.py]: just one last (small) thing, space between `/` and `5`

Comment (nikhilgeo) [inline modules/src/anime.py]: Sorry missed it..My Bad !!

**Comment by swapagarwal** [comment]
Thanks @nikhilgeo! :+1:

**Pull request merged**